packages/app/playwright/crx/crx.test.ts

```diff
@@ -338,8 +338,7 @@ test.describe('FuelWallet Extension', () => {
 
         // Confirm transaction
         await hasText(confirmTransactionPage, /0\.0000001.ETH/i);
-        await waitAriaLabel(confirmTransactionPage, senderAccount.name);
-        await getButtonByText(confirmTransactionPage, /confirm/i).click();
```

Comment:
The sender account name is essential in this test to ensure it shows the expected account on the approval screen.

Reply:
> But we don't show the account name on the approval screen, we only show the address. ![Screenshot 2023-02-27 at 4 27 36 PM](https://user-images.githubusercontent.com/15849997/221622301-169de1e0-8d8e-4cb6-933e-22f57395adfc.png) ![Screenshot 2023-02-27 at 4 27 40 PM](https://user-images.githubusercontent.com/15849997/221622305-f990f8ae-683c-437f-8fc9-e96eef1654df.png)

The test search for a aria-label, with the name of the account you can add it to the Account Details component, or change the test to check by the address.